src/qibo/tests_new/test_core_gates_density_matrix.py

```diff
@@ -68,7 +68,7 @@ def test_rygate_density_matrix(backend):
                          [("H", {}), ("X", {}), ("Y", {}), ("Z", {}), ("I", {}),
                           ("RX", {"theta": 0.123}), ("RY", {"theta": 0.123}),
                           ("RZ", {"theta": 0.123}), ("U1", {"theta": 0.123}),
-                          ("ZPow", {"theta": 0.123}),
+                          ("U1", {"theta": 0.123}),
```

Comment:
Similarly here we already test for U1 (see right above line) so no need to do this twice.